docs/faststore/docs/security/managing-secrets.mdx

```diff
@@ -0,0 +1,102 @@
+
+  %% 1. User initiates secret creation/update/deletion
+  User ->> WebOps: Request to create/update/delete secret
+
+  %% 2. WebOps validates permissions
+  WebOps ->> WebOps: Validate permissions and scope
+
+  %% 3. WebOps stores secret in AWS Secrets Manager
+  WebOps ->> AWS: Store/Update/Delete secret (with correct scope)
+  AWS -->> WebOps: Confirm secret storage/update/deletion
+
+  %% 4. WebOps determines which provider to use for build/deployment
+  WebOps ->> WebOps: Select configured provider
+
+  %% 5. WebOps propagates secret to selected provider and initiates build
+  alt Provider A
+    WebOps ->> ProviderA: Transmit secret securely
+    ProviderA ->> ProviderA: Consume secret, prepare environment
+    WebOps ->> ProviderA: Start build/deployment
+    ProviderA ->> ProviderA: Build and host application
+    ProviderA -->> WebOps: Send status update
+  else Provider B
+    WebOps ->> ProviderB: Transmit secret securely
+    ProviderB ->> ProviderB: Consume secret, prepare environment
+    WebOps ->> ProviderB: Start build/deployment
+    ProviderB ->> ProviderB: Build and host application
+    ProviderB -->> WebOps: Send status update
+  else Provider N
+    WebOps ->> ProviderN: Transmit secret securely
+    ProviderN ->> ProviderN: Consume secret, prepare environment
+    WebOps ->> ProviderN: Start build/deployment
+    ProviderN ->> ProviderN: Build and host application
+    ProviderN -->> WebOps: Send status update
+  end
+
+  %% 6. Feedback to user
+  WebOps -->> User: Notify build and secret operation success
+```
+
+When a user initiates a request to create or delete a secret, WebOps first validates the user's permissions and the request scope. The scope defines the context, such as project, environment, namespace, or account, that determines where and how the secret will be stored, propagated, and accessed.
+
+Once validated, WebOps forwards the operation to AWS Secrets Manager, which confirms the action. Next, WebOps identifies the appropriate deployment provider based on the project’s configuration. It then securely transmits the secret information to the selected provider.
+
+The chosen provider consumes the secret and prepares the environment, after which WebOps triggers the build and deployment process. The provider proceeds to build and host the application and returns a status update to WebOps.
+
+Finally, WebOps notifies the user that the secret operation and the deployment were successful.
+
+## Instructions
+
+To manage your secrets, access your [FastStore WebOps dashboard](https://developers.vtex.com/docs/guides/faststore/1-onboarding-dashboard) and navigate to the **Settings** tab.
+
+In the [Secrets](https://developers.vtex.com/docs/guides/faststore/1-onboarding-dashboard#secrets) section, you can [create](#creating-secrets), [update](#updating-secrets), or [delete](#deleting-secrets) secrets following the steps below.
+
+### Creating secrets
+
+1. Input the related values in the `Key` and `Value` fields.

```

Comment:
```suggestion
1. In the `Key` and `Value` fields, enter their respective values.

```

Reply:
Tem algum padrão para escrever esses valores? Seria possível adicionar algum exemplo fictício aqui?

Reply:
Acho que dá para explorar melhor o que são esses campos.
Por exemplo:

In the `Key` field, enter the name of the secret, which serves as its unique identifier (e.g., `VTEX_API_TOKEN`, `NEXT_SECRET_KEY`). In the `Value` field, enter the corresponding sensitive information you want to store (e.g., the actual token, key, or password).